### DISCUSSION TITLE
Wino Mail

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#17] [General]
# Wino Mail

Check if Wino Mail is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/a3b9d890-fe01-4a96-8ff5-ba9207ab2c00

**Comment by OpenSource03** [comment]
May work on ARM32 CPUs, but will not work on ARM64.

Up to date info: https://github.com/bkaankose/Wino-Mail/issues/97

**Reply by talynone** [reply]
They updated the app, it's now ARM64 native.

**Comment by talynone** [comment]
Just tested this from Windows Store, works just fine in emulation mode.

**Reply by OpenSource03** [reply]
Tested as well, works great. I've updated the status. Thanks.

**Reply by talynone** [reply]
They updated the app, it's now ARM64 native.

**Reply by OpenSource03** [reply]
Thanks! Updated. Note that status change may not reflect momentarily due to 2h edge caching.
